### PR TITLE
IO1-2949: ui issues additional changes

### PR DESCRIPTION
**Pull request by axelrod-blox**

### CODE REVIEW

File: src/app/components/applications/SSV/NewMyAccount/components/OperatorDashboard/OperatorDashboard.tsx [hunk 1]
```diff
@@ -76,7 +78,7 @@ const OperatorDashboard = () => {
   });
 
   const openSingleOperator = (listIndex: number, e: any) => {
-    if ( e.target === e.currentTarget.querySelector('.openExplorer') ){
+    if ( openExplorerRefs.includes(e.target) ){
```
Comment (meshin-blox) [inline src/app/components/applications/SSV/NewMyAccount/components/OperatorDashboard/OperatorDashboard.tsx:81]: Remove redundant spaces in condition

File: src/app/components/applications/SSV/NewMyAccount/components/changeOperatorName/ChangeOperatorName.tsx [hunk 2]
```diff
@@ -62,12 +62,12 @@ const ChangeOperatorName = () => {
     return (
         <BorderScreen
             blackHeader
-            header={'Change Operator Name'}
+            header={ 'Change Operator Name' }
```
Comment (meshin-blox) [inline src/app/components/applications/SSV/NewMyAccount/components/changeOperatorName/ChangeOperatorName.tsx:65]: If you take a look at all the { & } places in the code - there is no suffix and postfix spaces, lets keep the same approach everywhere

Comment (meshin-blox) [inline src/app/components/applications/SSV/NewMyAccount/components/changeOperatorName/ChangeOperatorName.tsx:65]: If you take a look at all the { & } places in the code - there is no suffix and postfix spaces, lets keep the same approach everywhere

Comment (meshin-blox) [inline src/app/components/applications/SSV/NewMyAccount/components/changeOperatorName/ChangeOperatorName.tsx:65]: If you take a look at all the { & } places in the code - there is no suffix and postfix spaces, lets keep the same approach everywhere

File: src/app/components/applications/SSV/RegisterValidatorHome/components/OfflineKeyShareGeneration/OfflineKeyShareGeneration.tsx [hunk 3]
```diff
@@ -31,10 +32,8 @@ const OfflineKeyShareGeneration = () => {
     const {
         operatorsIds,
         operatorsKeys,
-    } = Object.values(operatorStore.selectedOperators).sort((a: any, b: any) => a.id - b.id).reduce((aggr, operator: IOperator) => {
-        // @ts-ignore
+    } = Object.values(operatorStore.selectedOperators).sort((a: any, b: any) => a.id - b.id).reduce((aggr: any, operator: IOperator) => {
```
Comment (meshin-blox) [inline src/app/components/applications/SSV/RegisterValidatorHome/components/OfflineKeyShareGeneration/OfflineKeyShareGeneration.tsx:35]: This is weird part of the code, let's simplify it. First sort it to operators array constant. Then extract ids and keys in next code block

File: src/app/components/common/Button/CopyButton/CopyButton.tsx [hunk 4]
```diff
@@ -0,0 +1,19 @@
+import Grid from '@mui/material/Grid';
+import Typography from '@mui/material/Typography';
+import React from 'react';
```
Comment (meshin-blox) [inline src/app/components/common/Button/CopyButton/CopyButton.tsx:3]: Import React should be very first import in any tsx file

File: src/app/components/common/Button/CopyButton/CopyButton.tsx [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+type CopyButtonProps = {
+    textCopied: boolean;
+    classes: any;
+    onClickHandler: any;
+};
```
Comment (meshin-blox) [inline src/app/components/common/Button/CopyButton/CopyButton.tsx:9]: Need empty line between type and component definition

File: src/app/components/common/Button/CopyButton/CopyButton.tsx [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+export const CopyButton = ({ textCopied, classes, onClickHandler }: CopyButtonProps) => {
+    if (!textCopied) {
+        return <Grid item className={classes.CopyButton} onClick={onClickHandler}>Copy</Grid>;
+    }
+    return <Grid onClick={onClickHandler} container item className={classes.ButtonCopied}>
```
Comment (meshin-blox) [inline src/app/components/common/Button/CopyButton/CopyButton.tsx:14]: Wrap this greed with `(` and `)`

File: src/app/components/common/ImageDiv/ImageDiv.tsx [hunk 7]
```diff
@@ -8,15 +8,20 @@ type ImageDivProps = {
...
 
 const ImageDiv = (props: ImageDivProps) => {
-    const { onClick, image, width, height } = props;
+    const { onClick, image, width, height, setOpenExplorerRefs } = props;
     const classes = useStyles({ image, width, height });
+    const ref = useRef(null);
+
+    useEffect(() => {
+        setOpenExplorerRefs && setOpenExplorerRefs((state: any) => [...state, ref.current]);
+        }, []);
```
Comment (meshin-blox) [inline src/app/components/common/ImageDiv/ImageDiv.tsx:21]: 1. Use normal `if` statement here
2. Format code block properly

File: src/app/components/applications/SSV/RegisterValidatorHome/components/SelectOperators/components/FirstSquare/components/OperatorDetails/OperatorDetails.tsx [hunk 8]
```diff
@@ -15,10 +15,11 @@ type Props = {
   withCopy?: boolean;
   withoutExplorer?: boolean;
   operator: any // ?? IOperator
```
Comment (meshin-blox) [inline src/app/components/applications/SSV/RegisterValidatorHome/components/SelectOperators/components/FirstSquare/components/OperatorDetails/OperatorDetails.tsx:17]: What does this comment means? If it should be IOperator - use this type

File: src/app/components/applications/SSV/RegisterValidatorHome/components/OfflineKeyShareGeneration/OfflineKeyShareGeneration.tsx [hunk 9]
```diff
@@ -14,6 +14,7 @@ import ProcessStore from '~app/common/stores/applications/SsvWeb/Process.store';
 import NewWhiteWrapper from '~app/components/common/NewWhiteWrapper/NewWhiteWrapper';
 import NotificationsStore from '~app/common/stores/applications/SsvWeb/Notifications.store';
 import OperatorStore, { IOperator } from '~app/common/stores/applications/SsvWeb/Operator.store';
+import { CopyButton } from '~app/components/common/Button/CopyButton/CopyButton';
```
Comment (meshin-blox) [inline src/app/components/applications/SSV/RegisterValidatorHome/components/OfflineKeyShareGeneration/OfflineKeyShareGeneration.tsx:17]: In all the files that you touched sort imports using these rules:

1. Sort all third-party imports from shorter import to longer
2. Sort all the app-specific imports from shorter to longer
3. Importing React should be always first regardless length and type of import

**Review by meshin-blox — CHANGES_REQUESTED** [review]

File: src/app/components/applications/SSV/RegisterValidatorHome/components/OfflineKeyShareGeneration/OfflineKeyShareGeneration.tsx (hunk 9 above)
Comment (axelrod-blox) [reply src/app/components/applications/SSV/RegisterValidatorHome/components/OfflineKeyShareGeneration/OfflineKeyShareGeneration.tsx]: done

File: src/app/components/applications/SSV/RegisterValidatorHome/components/SelectOperators/components/FirstSquare/components/OperatorDetails/OperatorDetails.tsx (hunk 8 above)
Comment (axelrod-blox) [reply src/app/components/applications/SSV/RegisterValidatorHome/components/SelectOperators/components/FirstSquare/components/OperatorDetails/OperatorDetails.tsx]: done

File: src/app/components/common/ImageDiv/ImageDiv.tsx (hunk 7 above)
Comment (axelrod-blox) [reply src/app/components/common/ImageDiv/ImageDiv.tsx:23]: done

File: src/app/components/common/Button/CopyButton/CopyButton.tsx (hunk 6 above)
Comment (axelrod-blox) [reply src/app/components/common/Button/CopyButton/CopyButton.tsx]: done

File: src/app/components/applications/SSV/RegisterValidatorHome/components/OfflineKeyShareGeneration/OfflineKeyShareGeneration.tsx (hunk 3 above)
Comment (axelrod-blox) [reply src/app/components/applications/SSV/RegisterValidatorHome/components/OfflineKeyShareGeneration/OfflineKeyShareGeneration.tsx]: done

File: src/app/components/common/Button/CopyButton/CopyButton.tsx (hunk 5 above)
Comment (axelrod-blox) [reply src/app/components/common/Button/CopyButton/CopyButton.tsx:9]: done

File: src/app/components/common/Button/CopyButton/CopyButton.tsx (hunk 6 above)
Comment (axelrod-blox) [reply src/app/components/common/Button/CopyButton/CopyButton.tsx]: done

File: src/app/components/applications/SSV/NewMyAccount/components/changeOperatorName/ChangeOperatorName.tsx (hunk 2 above)
Comment (axelrod-blox) [reply src/app/components/applications/SSV/NewMyAccount/components/changeOperatorName/ChangeOperatorName.tsx]: done

File: src/app/components/common/Button/CopyButton/CopyButton.tsx (hunk 4 above)
Comment (axelrod-blox) [reply src/app/components/common/Button/CopyButton/CopyButton.tsx]: done

File: src/app/components/applications/SSV/NewMyAccount/components/changeOperatorName/ChangeOperatorName.tsx (hunk 2 above)
Comment (axelrod-blox) [reply src/app/components/applications/SSV/NewMyAccount/components/changeOperatorName/ChangeOperatorName.tsx]: done

Comment (axelrod-blox) [reply src/app/components/applications/SSV/NewMyAccount/components/changeOperatorName/ChangeOperatorName.tsx]: done

File: src/app/components/applications/SSV/NewMyAccount/components/OperatorDashboard/OperatorDashboard.tsx (hunk 1 above)
Comment (axelrod-blox) [reply src/app/components/applications/SSV/NewMyAccount/components/OperatorDashboard/OperatorDashboard.tsx]: done

File: src/app/components/applications/SSV/MyAccount/components/ClusterDashboard/ClusterDashboard.tsx [hunk 10]
```diff
@@ -5,7 +5,7 @@ import { useNavigate } from 'react-router-dom';
 import React, { useRef, useState } from 'react';
 import config from '~app/common/config';
 import { useStores } from '~app/hooks/useStores';
-import { useStyles } from '../../NewMyAccount.styles';
+import { useStyles } from '../../MyAccount.styles';
```
Comment (meshin-blox) [inline src/app/components/applications/SSV/MyAccount/components/ClusterDashboard/ClusterDashboard.tsx:8]: Use full path starting from `~app`, don't forget about sorting imports rule

File: src/app/components/applications/SSV/MyAccount/components/Dashboard/Dashboard.tsx [hunk 11]
```diff
@@ -114,8 +114,8 @@ const Dashboard = (props: DashboardProps) => {
           <TableBody>
             {rows.map((row, index: number) => (
                 <CustomizedBodyRow
-                    key={index}
-                    onClick={(e : any)=>{rowsAction && rowsAction(index, e);}}
+                    key={ index }
+                    onClick={( e: any )=>{rowsAction && rowsAction(index, e);}}
```
Comment (meshin-blox) [inline src/app/components/applications/SSV/MyAccount/components/Dashboard/Dashboard.tsx:118]: Remove unnecessary empty suffix and postfix

File: src/app/components/applications/SSV/MyAccount/components/OperatorDashboard/OperatorDashboard.tsx [hunk 12]
```diff
@@ -6,17 +6,17 @@ import React, { useEffect, useState } from 'react';
 import config from '~app/common/config';
 import { useStores } from '~app/hooks/useStores';
 import Status from '~app/components/common/Status';
-import { useStyles } from '../../NewMyAccount.styles';
+import { useStyles } from '../../MyAccount.styles';
```
Comment (meshin-blox) [inline src/app/components/applications/SSV/MyAccount/components/OperatorDashboard/OperatorDashboard.tsx:9]: Full path

File: src/app/components/applications/SSV/MyAccount/components/Withdraw/Withdraw.tsx [hunk 13]
```diff
@@ -2,7 +2,7 @@ import React, { useEffect } from 'react';
 import { observer } from 'mobx-react';
 import Grid from '@mui/material/Grid';
 import { useStores } from '~app/hooks/useStores';
-import { useStyles } from './NewWithdraw.styles';
+import { useStyles } from './Withdraw.styles';
```
Comment (meshin-blox) [inline src/app/components/applications/SSV/MyAccount/components/Withdraw/Withdraw.tsx:5]: Full path

File: src/app/components/applications/SSV/MyAccount/components/Withdraw/Withdraw.tsx [hunk 14]
```diff
@@ -2,7 +2,7 @@ import React, { useEffect } from 'react';
...
 import { formatNumberToUi } from '~lib/utils/numbers';
 import ValidatorFlow from './components/ValidatorFlow';
```
Comment (meshin-blox) [inline src/app/components/applications/SSV/MyAccount/components/Withdraw/Withdraw.tsx:7]: Full path

File: src/app/components/applications/SSV/MyAccount/components/Withdraw/components/OperatorFlow/OperatorFlow.tsx [hunk 15]
```diff
@@ -2,7 +2,7 @@ import { observer } from 'mobx-react';
 import Grid from '@mui/material/Grid';
 import React, { useState } from 'react';
 import { useStores } from '~app/hooks/useStores';
-import { useStyles } from '../../NewWithdraw.styles';
+import { useStyles } from '../../Withdraw.styles';
```
Comment (meshin-blox) [inline src/app/components/applications/SSV/MyAccount/components/Withdraw/components/OperatorFlow/OperatorFlow.tsx:5]: Full path

File: src/app/components/applications/SSV/MyAccount/components/Withdraw/components/ValidatorFlow/ValidatorFlow.tsx [hunk 16]
```diff
@@ -4,7 +4,7 @@ import { useNavigate } from 'react-router-dom';
 import React, { useEffect, useState } from 'react';
 import Validator from '~lib/api/Validator';
 import { useStores } from '~app/hooks/useStores';
-import { useStyles } from '../../NewWithdraw.styles';
+import { useStyles } from '../../Withdraw.styles';
```
Comment (meshin-blox) [inline src/app/components/applications/SSV/MyAccount/components/Withdraw/components/ValidatorFlow/ValidatorFlow.tsx:7]: Full path

File: src/app/components/applications/SSV/MyAccount/components/changeOperatorName/ChangeOperatorName.tsx [hunk 17]
```diff
@@ -67,7 +67,7 @@ const ChangeOperatorName = () => {
                 (
                     <Grid container item>
                         <Grid container style={{ gap: 24 }}>
-                            <Grid item className={classes.Text}>
+                            <Grid item className={ classes.Text }>
```
Comment (meshin-blox) [inline src/app/components/applications/SSV/MyAccount/components/changeOperatorName/ChangeOperatorName.tsx:70]: Remove empty spaces

File: src/app/components/applications/SSV/OperatorSuccessPage/OperatorSuccessPage.tsx [hunk 18]
```diff
@@ -11,7 +11,7 @@ import HeaderSubHeader from '~app/components/common/HeaderSubHeader';
 import PrimaryButton from '~app/components/common/Button/PrimaryButton';
 import OperatorStore from '~app/common/stores/applications/SsvWeb/Operator.store';
 import ApplicationStore from '~app/common/stores/applications/SsvWeb/Application.store';
-import OperatorId from '~app/components/applications/SSV/NewMyAccount/components/OperatorId';
+import OperatorId from '~app/components/applications/SSV/MyAccount/components/OperatorId';
 import LinkText from '~app/components/common/LinkText/LinkText';
```
Comment (meshin-blox) [inline src/app/components/applications/SSV/OperatorSuccessPage/OperatorSuccessPage.tsx:15]: Sort imports

File: src/app/components/common/ImageDiv/ImageDiv.tsx [hunk 19]
```diff
@@ -8,15 +8,22 @@ type ImageDivProps = {
...
-    const { onClick, image, width, height } = props;
+    const { onClick, image, width, height, setOpenExplorerRefs } = props;
     const classes = useStyles({ image, width, height });
+    const ref = useRef(null);
+
+    useEffect(() => {
+        if (setOpenExplorerRefs) {
+            setOpenExplorerRefs((state: any) => [...state, ref.current]);
+            }
+        }, []);
```
Comment (meshin-blox) [inline src/app/components/common/ImageDiv/ImageDiv.tsx:23]: Code block should be properly formatted (normalized block spacing)

**Review by meshin-blox — CHANGES_REQUESTED** [review]

File: src/app/components/applications/SSV/MyAccount/components/ClusterDashboard/ClusterDashboard.tsx (hunk 10 above)
Comment (axelrod-blox) [reply src/app/components/applications/SSV/MyAccount/components/ClusterDashboard/ClusterDashboard.tsx:8]: done

File: src/app/components/applications/SSV/MyAccount/components/Dashboard/Dashboard.tsx (hunk 11 above)
Comment (axelrod-blox) [reply src/app/components/applications/SSV/MyAccount/components/Dashboard/Dashboard.tsx:118]: done

File: src/app/components/applications/SSV/MyAccount/components/Withdraw/Withdraw.tsx (hunk 13 above)
Comment (axelrod-blox) [reply src/app/components/applications/SSV/MyAccount/components/Withdraw/Withdraw.tsx:5]: done

File: src/app/components/applications/SSV/MyAccount/components/Withdraw/Withdraw.tsx (hunk 14 above)
Comment (axelrod-blox) [reply src/app/components/applications/SSV/MyAccount/components/Withdraw/Withdraw.tsx:7]: done

File: src/app/components/applications/SSV/MyAccount/components/Withdraw/components/OperatorFlow/OperatorFlow.tsx (hunk 15 above)
Comment (axelrod-blox) [reply src/app/components/applications/SSV/MyAccount/components/Withdraw/components/OperatorFlow/OperatorFlow.tsx:5]: done

File: src/app/components/applications/SSV/MyAccount/components/Withdraw/components/ValidatorFlow/ValidatorFlow.tsx (hunk 16 above)
Comment (axelrod-blox) [reply src/app/components/applications/SSV/MyAccount/components/Withdraw/components/ValidatorFlow/ValidatorFlow.tsx:7]: done

File: src/app/components/applications/SSV/MyAccount/components/changeOperatorName/ChangeOperatorName.tsx (hunk 17 above)
Comment (axelrod-blox) [reply src/app/components/applications/SSV/MyAccount/components/changeOperatorName/ChangeOperatorName.tsx:70]: done

File: src/app/components/applications/SSV/OperatorSuccessPage/OperatorSuccessPage.tsx (hunk 18 above)
Comment (axelrod-blox) [reply src/app/components/applications/SSV/OperatorSuccessPage/OperatorSuccessPage.tsx:15]: done

File: src/app/components/common/ImageDiv/ImageDiv.tsx (hunk 19 above)
Comment (axelrod-blox) [reply src/app/components/common/ImageDiv/ImageDiv.tsx]: done

File: src/app/components/applications/SSV/MyAccount/components/Validator/EditFlow/ConfirmOperatorsChange/ConfirmOperatorsChange.tsx [hunk 20]
```diff
@@ -9,8 +9,8 @@ import BackNavigation from '~app/components/common/BackNavigation';
 import OperatorStore from '~app/common/stores/applications/SsvWeb/Operator.store';
 import MyAccountStore from '~app/common/stores/applications/SsvWeb/MyAccount.store';
 import ProcessStore, { SingleCluster } from '~app/common/stores/applications/SsvWeb/Process.store';
-import ValidatorWhiteHeader from '~app/components/applications/SSV/NewMyAccount/common/ValidatorWhiteHeader';
-import OperatorsReceipt from '~app/components/applications/SSV/NewMyAccount/components/Validator/EditFlow/OperatorsRecipt';
+import ValidatorWhiteHeader from '~app/components/applications/SSV/MyAccount/common/ValidatorWhiteHeader';
+import OperatorsReceipt from '~app/components/applications/SSV/MyAccount/components/Validator/EditFlow/OperatorsRecipt';
 import { useStyles } from './ConfirmOperatorsChange.styles';
```
Comment (meshin-blox) [inline src/app/components/applications/SSV/MyAccount/components/Validator/EditFlow/ConfirmOperatorsChange/ConfirmOperatorsChange.tsx:14]: Full path

File: src/app/components/applications/SSV/MyAccount/components/OperatorDashboard/OperatorDashboard.tsx (hunk 12 above)
Comment (meshin-blox) [inline src/app/components/applications/SSV/MyAccount/components/OperatorDashboard/OperatorDashboard.tsx:9]: Full path

File: src/app/components/applications/SSV/MyAccount/components/Validator/EditFlow/OperatorsRecipt/OperatorsReceipt.tsx [hunk 21]
```diff
@@ -20,7 +20,7 @@ import { addNumber, formatNumberToUi } from '~lib/utils/numbers';
 import { IOperator } from '~app/common/stores/applications/SsvWeb/Operator.store';
 import ValidatorStore from '~app/common/stores/applications/SsvWeb/Validator.store';
 import ApplicationStore from '~app/common/stores/applications/SsvWeb/Application.store';
-import RemainingDays from '~app/components/applications/SSV/NewMyAccount/common/RemainingDays';
+import RemainingDays from '~app/components/applications/SSV/MyAccount/common/RemainingDays';
 import OperatorDetails
   from '~app/components/applications/SSV/RegisterValidatorHome/components/SelectOperators/components/FirstSquare/components/OperatorDetails';
 import { useStyles } from './OperatorsReceipt.style';
```
Comment (meshin-blox) [inline src/app/components/applications/SSV/MyAccount/components/Validator/EditFlow/OperatorsRecipt/OperatorsReceipt.tsx:26]: Full path

File: src/app/components/applications/SSV/OperatorSuccessPage/OperatorSuccessPage.tsx [hunk 22]
```diff
@@ -7,12 +7,13 @@ import config from '~app/common/config';
 import { useStores } from '~app/hooks/useStores';
 import { useStyles } from './OperatorSuccessPage.styles';
```
Comment (meshin-blox) [inline src/app/components/applications/SSV/OperatorSuccessPage/OperatorSuccessPage.tsx:8]: Full path

File: src/app/components/applications/SSV/RegisterValidatorHome/components/FundingPeriod/FundingPeriod.tsx [hunk 23]
```diff
@@ -30,7 +30,7 @@ const FundingPeriod = () => {
   const ssvStore: SsvStore = stores.SSV;
   const processStore: ProcessStore = stores.Process;
   const operatorStore: OperatorStore = stores.Operator;
-  const [customPeriod, setCustomPeriod] = useState(730);
+  const [customPeriod, setCustomPeriod] = useState(config.GLOBAL_VARIABLE.DEFAULT_CLUSTER_PERIOD);
   const [checkedOption, setCheckedOption] = useState(options[1]);
   const timePeriodNotValid = customPeriod < 30;
```
Comment (meshin-blox) [inline src/app/components/applications/SSV/RegisterValidatorHome/components/FundingPeriod/FundingPeriod.tsx:35]: This also should be in a constant, like, `CLUSTER_VALIDITY_PERIOD_MINIMUM`

File: src/app/components/common/ImageDiv/ImageDiv.tsx [hunk 24]
```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import React, { useEffect, useRef } from 'react';
 import Grid from '@mui/material/Grid';
 import { observer } from 'mobx-react';
 import { useStyles } from './imageDiv.styles';
```
Comment (meshin-blox) [inline src/app/components/common/ImageDiv/ImageDiv.tsx:4]: Full path. Also style file should start with Uppercase

**Review by meshin-blox — CHANGES_REQUESTED** [review]

File: src/app/components/applications/SSV/RegisterValidatorHome/components/FundingPeriod/FundingPeriod.tsx (hunk 23 above)
Comment (axelrod-blox) [reply src/app/components/applications/SSV/RegisterValidatorHome/components/FundingPeriod/FundingPeriod.tsx:35]: done

File: src/app/components/applications/SSV/MyAccount/components/Validator/EditFlow/ConfirmOperatorsChange/ConfirmOperatorsChange.tsx (hunk 20 above)
Comment (axelrod-blox) [reply src/app/components/applications/SSV/MyAccount/components/Validator/EditFlow/ConfirmOperatorsChange/ConfirmOperatorsChange.tsx:14]: done

File: src/app/components/applications/SSV/MyAccount/components/Validator/EditFlow/OperatorsRecipt/OperatorsReceipt.tsx (hunk 21 above)
Comment (axelrod-blox) [reply src/app/components/applications/SSV/MyAccount/components/Validator/EditFlow/OperatorsRecipt/OperatorsReceipt.tsx:26]: done

File: src/app/components/applications/SSV/MyAccount/components/OperatorDashboard/OperatorDashboard.tsx (hunk 12 above)
Comment (axelrod-blox) [reply src/app/components/applications/SSV/MyAccount/components/OperatorDashboard/OperatorDashboard.tsx:9]: done

File: src/app/components/common/ImageDiv/ImageDiv.tsx (hunk 24 above)
Comment (axelrod-blox) [reply src/app/components/common/ImageDiv/ImageDiv.tsx:4]: done

File: src/app/components/applications/SSV/MyAccount/components/OperatorDashboard/OperatorDashboard.tsx (hunk 12 above)
Comment (axelrod-blox) [reply src/app/components/applications/SSV/MyAccount/components/OperatorDashboard/OperatorDashboard.tsx:9]: done

**Review by meshin-blox — APPROVED** [review]

**Pull request merged**